pkg/systemd/services/timer-dialog.jsx

```diff
@@ -335,7 +335,8 @@ const CreateTimerDialogBody = ({ owner }) => {
                                                             const arr = [...repeatPatterns];
                                                             arr[idx].date = str;
                                                             setRepeatPatterns(arr);
-                                                        }} />
+                                                        }}
+                                                        appendTo={() => document.body} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17826-20221018-195623-2ba05daf-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)